package.json

```diff
@@ -40,20 +40,36 @@
     "node": ">=7.10.0"
   },
   "files": [
-    "lib",
-    "index.js",
-    "cli.js",
+    "dist/lib",
+    "dist/index.js",
+    "dist/cli.js",
+    "dlls",
     "package.json",
     "README.md",
     "LICENSE"
   ],
-  "main": "./index.js",
+  "main": "./dist/index.js",
   "bin": {
-    "oad": "./cli.js"
+    "oad": "./dist/cli.js"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "dist/**/*.js"
+    ],
+    "testMatch": [
+      "**/*test.js"
+    ]
   },
   "scripts": {
-    "jshint": "jshint index.js --reporter=jslint",
-    "test": "npm -s run-script jshint && mocha -t 50000",
-    "gulp": "gulp"
+    "dn.clean": "dotnet clean openapi-diff/OpenApiDiff.sln",
+    "dn.restore": "dotnet restore openapi-diff/OpenApiDiff.sln",
+    "dn.build": "npm run dn.clean && npm run dn.restore && dotnet build -c release openapi-diff/OpenApiDiff.sln /nologo /clp:NoSummary",
+    "dn.publish": "npm run dn.build && dotnet publish -c release openapi-diff/src/core/OpenApiDiff/OpenApiDiff.csproj",
+    "dn.test": "npm run dn.build && dotnet test openapi-diff/src/modeler/AutoRest.Swagger.Tests/AutoRest.Swagger.Tests.csproj",
+    "tsc": "tsc",
+    "ts.test": "tsc && jest",
```

Comment:
these are a little confusing, not sure which one i need to run to validate a change :)

Reply:
Always run `test` to test everything. Nothing is changed. :-)